api/v1beta1/ipaddresspool_types.go

```diff
@@ -39,6 +39,23 @@ type IPAddressPoolSpec struct {
 	// +optional
 	// +kubebuilder:default:=false
 	AvoidBuggyIPs bool `json:"avoidBuggyIPs,omitempty"`
+
+	// AllocateTo makes ip pool allocation to specific namespace and/or service.
```

Comment:
Please add also a comment on the behaviour (I think we decided the lower value, the higher the priority)

Reply:
sure, changed it as per this design [proposal](https://github.com/metallb/metallb/blob/main/design/pool-configuration.md#using-a-given-ipaddresspool-only-for-a-subset-of-services-1).